src/config_creature.c

```diff
@@ -612,6 +625,31 @@ TbBool parse_creaturetypes_common_blocks(char *buf, long len, const char *config
                     COMMAND_TEXT(cmd_num), block_buf, config_textname);
             }
             break;
+        case 7: // SWAPCREATURES
+            while (get_conf_parameter_single(buf, &pos, len, gameadd.crtr_conf.model[n + 1].name, COMMAND_WORD_LEN) > 0)
+            {
+                newcrtr_desc[n].name = gameadd.crtr_conf.model[n + 1].name;
+                newcrtr_desc[n].num = n + 1;
+                n++;
+                if (n + 1 >= CREATURE_TYPES_MAX)
```

Comment:
I believe this should be SWAP_CREATURE_TYPES_MAX instead of the normal one, also use that constant for the 128 up top, and check all new CREATURE_TYPES_MAX if they should SWAP_CREATURE_TYPES_MAX  instead